index.js

```diff
@@ -340,14 +339,14 @@ function ensureValidOptions(options) {
     if(!options) throw new Error("options are required by express-winston middleware");
     if(!((options.transports && (options.transports.length > 0)) || options.winstonInstance))
         throw new Error("transports or a winstonInstance are required by express-winston middleware");
+
+    if (options.dynamicMeta && !_.isFunction(options.dynamicMeta)) {
+        throw new Error("`dynamicMeta` express-winston option should be a function");
+    }
 }
 
 function ensureValidLoggerOptions(options) {
     if (options.ignoreRoute && !_.isFunction(options.ignoreRoute)) {
         throw new Error("`ignoreRoute` express-winston option should be a function");
     }
-
```

Comment:
Since the option to handle `dynamicMeta` in the `errorLogger` function was added on commit [dc87e35a](https://github.com/bithavoc/express-winston/pull/138/commits/dc87e35a0629f5faaeb2db5cbd95e2e1f939f6c6), this `dynamicMeta` validation was moved to the `ensureValidOptions` function.